code/controllers/configuration/entries/game_options.dm

```diff
@@ -444,3 +444,26 @@
 /datum/config_entry/flag/spare_enforce_coc
 
 /datum/config_entry/flag/station_traits
+
+//Dynamic options
+/datum/config_entry/number/dynamic_minimum_station_integrity
+	config_entry_value = 0.95
```

Comment:
Dynamic has its own config, simply add a var to `/datum/game_mode/dynamic` and it can be overriden with `dynamic.json`. remove this.

Reply:
Done.